site/assets/js/Pricing-0.4.0.js

```diff
@@ -0,0 +1,291 @@
+    this.hostingPriceDiv = document.getElementById('hosting-price');
+    this.planSelectionDiv = document.getElementById('plan-selection');
+    this.planPriceDiv = document.getElementById('plan-price');
+    this.sumDiv = document.getElementById('sum');
+    this.monthlyActiveUserSlider = document.querySelector('input[name=monthly-active-users]');
+    this.monthlyActiveUserSlider.addEventListener('input', event => this._handleSliderChange(event));
+    this.monthlyActiveUserSlider.addEventListener('mouseup', event => this._handleSliderChange(event));
+    this.monthlyActiveUserSliderMin = parseInt(this.monthlyActiveUserSlider.getAttribute('min'));
+    this.monthlyActiveUserSliderMax = parseInt(this.monthlyActiveUserSlider.getAttribute('max'));
+    this.monthlyActiveUserValue = document.getElementById('monthly-active-users-value');
+    this.communityButton = document.querySelector('a[data-plan=Community]');
+    this.starterButton = document.querySelector('a[data-plan=Starter]');
+    this.purchaseButton = document.getElementById('purchase-button');
+
+    this.yearlyRadio = document.getElementById('yearly_radio');
+    this.yearlyRadio.addEventListener('change', event => this._handleBillingIntervalChange(event));
+    this.monthlyRadio = document.getElementById('monthly_radio');
+    this.monthlyRadio.addEventListener('change', event => this._handleBillingIntervalChange(event));
+
+    this.hostingPrice = 0;
+    this.planPrice = 0;
+    this._loadState();
+
+    this.yearlyRadio.checked = true;
+    this.billingInterval = 'yearly';
+
+    document.querySelectorAll('a[data-step]')
+            .forEach(e => e.addEventListener('click', event => this._handleStepClick(event)))
+    document.querySelectorAll('a[data-plan]')
+            .forEach(e => e.addEventListener('click', event => this._handlePlanClick(event)))
+    window.addEventListener('popstate', event => this._handleStateChange(event));
+
+    fetch('https://account-local.fusionauth.io/ajax/purchase/price-model')
+        .then(response => response.json())
+        .then(json => {
+          this.priceModel = json;
+          this._changeStep();
+          this._drawPlanPrices();
+        })
+  }
+
+  _calculateHostingPrice(type) {
+    let price = 0;
+
+    if (type === 'basic-cloud') {
+      price = this.priceModel.ec2['medium'] / 2;
+    } else if (type === 'business-cloud') {
+      price = this.priceModel.ec2['medium'] + this.priceModel.rds['medium'];
+    } else if (type === 'ha-cloud') {
+      price = (2 * this.priceModel.ec2['medium']) + this.priceModel.elb.base + (this.priceModel.rds['medium'] * 2);
+    }
+
+    return price;
+  }
+
+  _getBillingIntervalKey() {
+    return 'pricePerUnit' + (this.billingInterval === 'yearly' ? 'Yearly' : 'Monthly');
+  }
+
+  _calculatePlanPrice(plan) {
+    if (plan === 'Community') {
+      return 0;
+    }
+
+    var mau = parseInt(this.monthlyActiveUserSlider.value);
```

Comment:
As we move towards ECMAScript 6, we should try and use `let` or `const` for variable declarations.